src/main/frontend/components/block.cljs

```diff
@@ -648,7 +648,7 @@
                                          (->>
                                           (for [page (string/split page-original-name #"/")]
                                             (when (and (string? page) page)
-                                              (page-reference false page {} nil)))
+                                              (page-reference false page {:preview? true} nil)))
```

Comment:
```suggestion
                                              ;; hovering on page ref in page-preview does not show another page-preview if {:preview? true} was passed to page-blocks-cp
                                              (page-reference false page {:preview? true} nil)))
```

Reply:
I have added some doc strings to `page-cp` and `page-reference`. I think it is better than adding a comment to the line where invokes `page-reference` with argument `{:preview? true}`.